wifite/args.py

```diff
@@ -443,6 +427,20 @@ def _add_wps_args(self, wps):
         wps.add_argument('-wpsto', help=argparse.SUPPRESS, action='store',
                 dest='wps_timeout_threshold', type=int)
 
+    def _add_pmkid_args(self, pmkid):
+        pmkid.add_argument('-pmkid', '--pmkid',
```

Comment:
That's a better way to create an alias I guess

Reply:
The single-dash options (like `-wep`) are artifacts from the old version of Wifite. I guess it's for lazy people that forget to type `--`

I hide "aliases" to keep the help doc simple & formatted. Otherwise, help will print `-pmkid, --pmkid` which sometimes overfills the column, and I feel it makes the help doc bloated/busy.